apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8/ChronoLocalDateAssertionsSpec.kt

```diff
@@ -1,4 +1,4 @@
-package ch.tutteli.atrium.api.fluent.en_GB.jdk8
+package ch.tutteli.atrium.api.fluent.en_GB
```

Comment:
here (and in all other files in the jdk8 extension) you should keep the package as is

Reply:
@assaflei already revert this (basically revert all files in apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8); 
Add the following to each file at the beginning instead:
```
//TODO remove file with 1.0.0
@file:Suppress("DEPRECATION")
```
I will look into the problem of the build tomorrow.